main.py

```diff
@@ -2,6 +2,52 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    for genre in genres:
+        Genre.objects.create(name=genre)
+
+    actors = [
+        ["George", "Klooney"],
+        ["Kianu", "Reaves"],
+        ["Scarlett", "Keegan"],
+        ["Will", "Smith"],
+        ["Jaden", "Smith"],
+        ["Scarlett", "Johansson"]
+    ]
+    for actor in actors:
+        first_name, last_name = actor
+        Actor.objects.create(first_name=first_name, last_name=last_name)
```

Comment:
You could perform unpacking during iteration
```suggestion
    for first_name, last_name in actors:
        Actor.objects.create(first_name=first_name, last_name=last_name)
```